checkbox-core-snap/series22/snap/snapcraft.yaml

```diff
@@ -273,6 +273,9 @@ parts:
       python3 manage.py validate
       python3 manage.py build
       python3 manage.py install --layout=relocatable --prefix=/providers/checkbox-provider-base --root="$SNAPCRAFT_PART_INSTALL"
+      if [ $SNAPCRAFT_TARGET_ARCH = "armhf" ]; then
```

Comment:
Please add a comment to explain why this is done

Reply:
- Is this happening on any other arch? if not, why? if it is, clean them as well
- What is this library and does it work after it is cleaned of the executable stack?
- Do we need this library? Can we just remove it?

Reply:
This patch is more a hotfix to get back the ability to upload snaps to the store. Do we need libde265-0? probably not, like many other dependencies added because of opencv. 
About the other architectures, they don't have the same problem. I've downloaded and extracted the library file from the [debian packages ](https://launchpad.net/ubuntu/+source/libde265) and indeed only armhf has this bit enabled.